config/seo.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Search Engine Optimization
+    |--------------------------------------------------------------------------
+    |
+    | This file is for storing the configuration values related to search engine optimization.
+    |
+    */
+
+    'sitemap' => [
+        'patterns' => '*.about.*',
+        'to_ignore' => [
+            '*.about.page',
+        ],
```

Comment:
This doesn't appear to be used. I suppose we actually wanted to use it to ignore the generic Page route.